File: Python/latihan/konstanta.py
from typing import Final
from pconst import const
import constant

PI: Final = 3.14
print("Phi: %.2f" % (PI))

const.FIRSTNAME = "Retno"
print(f"Hallo {const.FIRSTNAME}!")

print("Gravity:", constant.GRAVITY)


"""
    - Deklarasi konstanta di Python dilakukan menggunakan bantuan tipe class bernama typing.Final
    - Nama konstanta harus dituliskan dalam huruf besar(UPPER_CASE)
"""

File: C++/latihan/konstanta.cpp
#include <iostream>
#define PI 3.14 // menggunakan #define

using namespace std;

int main()
{
    // menggunakan keyword const
    const string FIRSTNAME = "Retno";

    // menggunakan keyword constexpr
    constexpr float GRAVITY = 9.8;

    cout << "Phi: " << PI << endl;
    cout << "Hello "<<FIRSTNAME << endl;
    cout << "Gravity: " << GRAVITY << endl;
}

/*
    - Ketika melakukan deklarasi konstanta harus diberi nilai
*/